<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RBDiet</class>
 <widget class="QDialog" name="RBDiet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="4" column="8">
    <widget class="QLineEdit" name="edtEndAgeCount">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inputMask">
      <string>000; </string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="2">
    <widget class="QLineEdit" name="edtBegAgeCount">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inputMask">
      <string>000; </string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="lblName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Наименование</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>&amp;Дата начала</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QComboBox" name="cmbEndAgeUnit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Д</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Н</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>М</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Г</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="5" colspan="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2" colspan="3">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QComboBox" name="cmbBegAgeUnit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Д</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Н</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>М</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Г</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QLabel" name="lblAgeSep">
     <property name="text">
      <string>по</string>
     </property>
     <property name="buddy">
      <cstring>cmbEndAgeUnit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="9">
    <spacer name="horizontalSpacer_9">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="lblAge">
     <property name="text">
      <string>В&amp;озраст</string>
     </property>
     <property name="buddy">
      <cstring>cmbBegAgeUnit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>103</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Д&amp;ата окончания</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lblCode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Код</string>
     </property>
     <property name="buddy">
      <cstring>edtCode</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="8">
    <widget class="QLineEdit" name="edtCode"/>
   </item>
   <item row="1" column="2" colspan="8">
    <widget class="QLineEdit" name="edtName"/>
   </item>
   <item row="6" column="0" colspan="10">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>cmbBegAgeUnit</tabstop>
  <tabstop>edtBegAgeCount</tabstop>
  <tabstop>cmbEndAgeUnit</tabstop>
  <tabstop>edtEndAgeCount</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RBDiet</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RBDiet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
